src/LandIce/problems/LandIce_StokesFOBase.hpp

```diff
@@ -546,9 +547,9 @@ constructVelocityEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
   ev = evalUtils.getPSTUtils().constructNodesToCellInterpolationEvaluator ("temperature",false);
   fm0.template registerEvaluator<EvalT> (ev);
 
-  // Interpolate ice softness (aka, flow_factor) from nodes to cell
-  ev = evalUtils.getPSTUtils().constructNodesToCellInterpolationEvaluator ("flow_factor",false);
-  fm0.template registerEvaluator<EvalT> (ev);
+//  // Interpolate ice softness (aka, flow_factor) from nodes to cell
```

Comment:
@bartgol I had to comment this because it conflicts with the FlowRate evaluator registered at line 646. Is that OK to comment this out, or should we put some logic to address this?

Reply:
You may be right. I think I added that interpolation because at some point the flow_factor was loaded from file at nodes. Maybe I was wrong, but the scenario was never tested... Anyhow, the way it is now should be fine.